contrib/epee/src/mlog.cpp

```diff
@@ -184,6 +184,28 @@ void mlog_set_log(const char *log)
   }
 }
 
+// %locname custom specifier can be used in addition to the Logging Format Specifiers of the Easylogging++
+// %locname similar to %loc but without full path
+//the default format is "%datetime{%Y-%M-%d %H:%m:%s.%g}	%thread	%level	%logger	%loc	%msg"
+void mlog_set_format(const char* format)
+{
+    auto locname = [](const el::LogMessage* lm)-> std::string
+    {
+        std::string s = lm->file();
```

Comment:
it's better to define different separators for linux and win

Reply:
Done.